### PR TITLE
Use @JsonFilter instead of Mix-In

### PR DESCRIPTION
**Pull request by dacrome**
To make sure Spring's object mapper is also usable
in the `AttributesRemovalHelper`, this PR get rid off
Jackson's Mix-In and introduce `@JsonFilter`.
This is only possible  by configuring on startup a
filter which does no property filtering.

### CODE REVIEW

**Comment by dacrome** [comment]
Hey @tkrille,
please run this change with the ITs, because you are the only one whose IT test order is showing that the current master is broken :)

File: src/main/java/org/osiam/resources/helper/AttributesRemovalHelper.java [hunk 1]
```diff
@@ -43,6 +55,11 @@
 
     private ObjectMapper mapper = new ObjectMapper();
```
Comment (tkrille) [inline src/main/java/org/osiam/resources/helper/AttributesRemovalHelper.java:56]: Remove the constructor call and make this field `private`.

File: src/main/java/org/osiam/resources/helper/AttributesRemovalHelper.java [hunk 2]
```diff
@@ -135,22 +152,21 @@
     }
 
     private ObjectWriter getObjectWriter(List<String> fieldsToReturn) {
+        if (fieldsToReturn.isEmpty()) {
+            FilterProvider filter = new SimpleFilterProvider().addFilter("filter properties by name", SimpleBeanPropertyFilter.serializeAll());
```
Comment (tkrille) [inline src/main/java/org/osiam/resources/helper/AttributesRemovalHelper.java]: Do we really need another no-op filter? Shouldn't the one defined in the global `ObjectMapper` be enough?

File: src/main/java/org/osiam/resources/helper/AttributesRemovalHelper.java (hunk 1 above)
Comment (dacrome) [inline src/main/java/org/osiam/resources/helper/AttributesRemovalHelper.java:56]: Thanks, good catch!

File: src/main/java/org/osiam/resources/helper/AttributesRemovalHelper.java (hunk 2 above)
Comment (dacrome) [inline src/main/java/org/osiam/resources/helper/AttributesRemovalHelper.java]: Yes, good catch again

**Comment by dacrome** [comment]
I close this PR because @wallner created a [better solution](https://github.com/osiam/osiam/pull/149).

**Pull request closed without merging**